datacenterlight/locale/de/LC_MESSAGES/django.po

```diff
@@ -284,20 +305,37 @@ msgstr "inkl. Mehrwertsteuer"
 msgid "Month"
 msgstr "Monat"
 
+msgid "Sign up"
+msgstr ""
+
+#, fuzzy
+#| msgid "Billing Address"
 msgid "Billing Address"
 msgstr "Rechnungsadresse"
 
+#, fuzzy
```

Comment:
I think, we should make a habit that we verify the fuzzy translations are not included in the PRs @tiwariav or if they are included we have a valid reason for that. The following for instance is a wrong translation.